MIGRATION_v2-to-v3.md

```diff
@@ -4,9 +4,13 @@ If you are using v2 and want to upgrade to v3, we have tried to keep the changes
 
 ## Common changes
 * Installation `npm i @sendbird/uikit-react`
-* There are no changes to SendbirdProvider, App, withSendbird, useSendbirdStateContext etc,
+* There are no changes to App, withSendbird, useSendbirdStateContext etc,
 * import path should start from `"@sendbird/uikit-react`!
 
+## SendbirdProvider
+### Changes
+  * `UserListQuery.next(callback: unknown): void;` changed to `next(): Promise<User[]>;`(chat v4 change)
```

Comment:
Do you mean [ApplicationUserListQuery](https://sendbird.com/docs/chat/v4/javascript/ref/classes/_sendbird_chat.ApplicationUserListQuery.html) here?